### PR TITLE
[WIP]Ignore env file to remove sensitive app variables from repo

### PR DESCRIPTION
**Pull request by pmcostello**
Make sure that .env is ignored.
Add instruction about Copying the .env-example file
Add instructions to Readme about CMS and Google

---
This change is [Reviewable](https://reviewable.io/reviews/moneyadviceservice/frontend/1689)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -60,6 +60,12 @@ Copy the features.yml.sample to the config dir:
 cp config/features.yml.sample config/features.yml
 ```
 
+Make sure to copy the .env.example file:				
+```sh		
+cp .env.example .env
+
```
Comment (norm) [inline README.md]: You need to close the code block you've opened there.

File: README.md [hunk 2]
```diff
@@ -70,7 +76,7 @@ $ foreman s
 
 ### Change CMS URL Path
 
-In development, frontend will use the QA CMS for convenience. You can change the MAS_CMS_URL on [.env](https://github.com/moneyadviceservice/frontend/blob/master/.env#L2) file. Use http://comfy.moneyadviceservice.org.uk to point to LIVE content. Or http://localhost:PORT to point to a local running CMS.
+In development, frontend will use the local CMS for convenience. You can change the MAS_CMS_URL on [.env] file. Use http://cms.moneyadviceservice.org.uk to point to LIVE content, https://cms.qa.dev.mas.local for testing Or http://localhost:PORT to point to a local running CMS.
```
Comment (norm) [inline README.md]: You've removed the destination of the link for `[.env]`, but left it as a link.

**Comment by norm** [comment]
Probably should also add `.env` to `.gitignore` to stop this happening again.

**Comment by pmcostello** [comment]
darn, I thought I had added it to .gitignore, must've reverted it before pushing, nice spot, thanks :)

**Comment by jongilbraith** [comment]
👍

**Comment by Tr4pSt3R** [comment]
👍

File: .env-example [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+STATSD_HOST=localhost
+STATSD_PORT=8125
+WEBCHAT_DOMAIN=xxxx.xxxx
+
+CAR_COST_TOOL_DOMAIN=localhost
+CAR_COST_TOOL_CAP_SUBSCRIBER_ID=''
+CAR_COST_TOOL_CAP_PASSWORD=''
+CAR_COST_TOOL_MAX_CARS_TO_COMPARE=3
+AUTH_USERNAME=admin
+AUTH_PASSWORD=password
```
Comment (tomas-stefano) [inline .env-example]: I am not sure if it is related to this PR, but we are not using MAS_CONTENT_SERVICE_URL (Hippo CMS) anymore.

Proof: 
https://github.com/search?q=org%3Amoneyadviceservice+MAS_CONTENT_SERVICE_URL&type=Code

Interestly we still have on the code the MAS_PUBLIC_WEBSITE_URL (old site), but there is only a public website assignment of the variable but not being used:

https://github.com/search?utf8=%E2%9C%93&q=org%3Amoneyadviceservice+MAS_PUBLIC_WEBSITE_URL&type=Code&ref=searchresults
https://github.com/moneyadviceservice/frontend/blob/473919b720d5d0e8b4286b595b3b746fc4a06ddd/config/initializers/core.rb#L9
And the public_website_connection variable is not being used in anywhere on the app besides the setup: https://github.com/search?utf8=%E2%9C%93&q=org%3Amoneyadviceservice+public_website_connection&type=Code&ref=searchresults

Wdyt that we should be actioning for this env vars?

Comment (pmcostello) [inline .env-example]: the MAS_CONTENT_SERVICE_URL can be removed but we need to keep the MAS_PUBLIC_WEBSITE_URL or rename it as the tests on frontend fail locally for me without it

Comment (pmcostello) [inline .env-example]: i've removed the MAS_CONTENT_SERVICE_URL from the repo, i'll close and merge this PR now unless anyone has any objections?

**Pull request merged**